pkg/dns/gcp/provider.go

```diff
@@ -2,9 +2,8 @@ package gcp
 
 import (
 	"context"
-	"net/http"
-
 	"google.golang.org/api/googleapi"
+	"net/http"
```

Comment:
Standard go libraries should be imported first, so I would undo this change.

Reply:
the go fmt does not allow this.

Reply:
If you add the blank line back, sorting shouldn't matter. ie:
```suggestion
	"context"		"context"
	"net/http"	

	"google.golang.org/api/googleapi"
```